lib/brcobranca/remessa/cnab240/base.rb

```diff
@@ -282,6 +316,8 @@ def monta_lote(nro_lote)
             contador += 1
             lote << monta_segmento_q(pagamento, nro_lote, contador)
             contador += 1
+            lote << monta_segmento_r(pagamento, nro_lote, contador)
```

Comment:
massa @marceloboth ! acho que seria interessante adicionar então apenas quando for desenvolvida a funcionalidade da remessa para Cecred, uma vez que agora não teria porque remover o segmento. O que acham?